apps/webservice/src/app/[workspaceSlug]/systems/[systemSlug]/deployments/ReleaseDropdownMenu.tsx

```diff
@@ -0,0 +1,175 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+import { TbAlertTriangle, TbDotsVertical, TbReload } from "react-icons/tb";
+
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from "@ctrlplane/ui/alert-dialog";
+import { Badge } from "@ctrlplane/ui/badge";
+import { Button, buttonVariants } from "@ctrlplane/ui/button";
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@ctrlplane/ui/dialog";
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from "@ctrlplane/ui/dropdown-menu";
+
+import { api } from "~/trpc/react";
+
+const RedeployReleaseDialog: React.FC<{
+  release: {
+    id: string;
+    name: string;
+  };
+  environment: {
+    id: string;
+    name: string;
+  };
```

Comment:
```suggestion
  release: {
    id: string;
    name: string;
  };
  environment: {
    id: string;
    name: string;
  };
```

I think you can one liner this :)